mobile_config/src/cli/daemon.rs

```diff
@@ -0,0 +1,93 @@
+use crate::{
+    admin_service::AdminService,
+    authorization_service::AuthorizationService,
+    carrier_service::CarrierService,
+    entity_service::EntityService,
+    gateway::{service::GatewayService, tracker::Tracker},
+    grpc_server::GrpcServer,
+    hex_boosting_service::HexBoostingService,
+    key_cache::KeyCache,
+    settings::Settings,
+    sub_dao_service::SubDaoService,
+};
+
+use task_manager::TaskManager;
+
+#[derive(Debug, clap::Args)]
+pub struct Daemon;
```

Comment:
If we're going to follow the pattern, can we rename this to `Server`?

It seems weird to have the cli command be `server` that delegates to a file called `deamon.rs`.

Reply:
👍🏻